src/main/git-elegant-completion

```diff
@@ -17,6 +17,11 @@ _git_elegant() {
         configure)
             COMPREPLY=($(compgen -W '--global --local' -- ${cur}) )
             return 0 ;;
+        check)
+            local data=("-a" "--all" "-u" "--unstaged" "-s" "--staged")
```

Comment:
The completion list is not correct. Please pay attention to the description of #12.

Reply:
@extsoft Yes, sorry. I will correct it.